app/assets/javascripts/course/user_achievements.js

```diff
@@ -0,0 +1,12 @@
+(function($) {
+  'use strict';
+  $('#toggle-on-button' ).click(function(e) {
```

Comment:
If you're also feeling brave, this component can be reused in multiple places. Extract it out as a component.